packages/@react-spectrum/tag/src/TagGroup.tsx

```diff
@@ -101,6 +86,7 @@ function TagGroup<T extends object>(props: SpectrumTagGroupProps<T>, ref: DOMRef
           styleProps.className
         )
       }
+      role={state.collection.size ? 'grid' : null}
```

Comment:
seems like this should be logic inside `useGrid`?

Reply:
I did this to prevent the role assignment when there are no children of 'grid' for accessibility. For example:
```
let items = []
<TagGroup items={items}>
  // no children but the group is still rendered
  item => (<Item>{item.label}</Item>)
</TagGroup>

Reply:
Thanks for the explanation. This is fine for now. I'll follow up with Design to see if we need to add something for empty state like we do for ListView and other collections

Reply:
I've talked to design, we're going to go the render absolutely nothing route for "empty" taggroups. This will be done in a followup PR.